### ISSUE TITLE
Unclear title "Color mode: Default" under File > Export as... > PDF

### ISSUE THREAD

**Issue by reinisberzins** [OpenOrienteering/mapper#1664]
### Steps to reproduce
1. File>Export as... > PDF > Mode: Vector graphics
2. I leave "Color mode: Default" not being aware that it's RGB which isn't suitable for printing. I would recommend changing the title "Default" (which doesn't tell anything) to something like "RGB (for screens)" and "Device CMYK" (some might even misunderstand that "device" mean smartphone/tablet) to something like "CMYK (for printing)". It's just my idea to make it more understandable for inexperienced users like me.

### Actual behaviour
Exported PDF file has changed CMYK colours, e.g.:
Green 60% color layer (for symbol 408 Vegation: walk) should correspond to CMYK colour: Cyan=42 Yellow=79
Yellow 100%/Green 50% colour layer (for symbol 420 Area that shall not be entered) should correspond to CMYK colour: Cyan=44 Magenta=10 Yellow=100

### Expected behaviour
Original colours in the .omap file were:
Green 60% color layer (for symbol 408 Vegation: walk) should correspond to CMYK colour: Cyan=46 Yellow=55
Yellow 100%/Green 50% colour layer (for symbol 420 Area that shall not be entered) should correspond to CMYK colour: C=38 M=27 Y=100

### Configuration
Mapper Version: 0.9.3
Operating System: Windows 10 x64

**Comment by dg0yt** [comment]
The first option is called "default" because it does what you would normally get from an application being based on Qt. And this happens to be RGB.

The other option is based on code I added to a fork of Qt's PDF code. It outputs the map drawing in a color space called "Device CMYK". The results are still very much dependent on the actual device (colorants). Note that all (at least most) templates are still output in RGB. 

RGB could even be more accurate than CMYK if ... it was attached to a proper color space. Unfortunately, we only get "DeviceRGB",

**Comment by dg0yt** [comment]
> Green 60% color layer (for symbol 408 Vegation: walk) originally is Cyan=46 Yellow=55, but changes to Cyan=42 Yellow=79 in PDF file.
Yellow 100%/Green 50% colour layer (for symbol 420 Area that shall not be entered) originally is C=38 M=27 Y=100, but changes to Cyan=44 Magenta=10 Yellow=100 in PDF file.

If you refer to output in "Device CMYK" mode, it is a bug. If you refer to output in RGB, than the report is invalid: There is no CMYK in the PDF file, just RGB.

**Comment by Symbian9** [comment]
> The first option is called "default" because it does what you would normally get from an application being based on Qt.

Not all users know what is "normally get from an application being based on Qt", so I agree with request to rename ~~*"Default"*~~ to *"Device RGB (default)"*.

**Comment by reinisberzins** [comment]
The described colour change happened when exporting File > Export as... > PDF > Color mode: **Default**. I analysed the resulting colours with an eyedropper tool - they had changed significantly. Also when printed them with 2 different printers - colours didn't look like on the screen and were hardly distinguishable from each other.
When I chose Color mode: **Device CMYK** instead the colours were exactly the same in the resulting PDF file and also the printed versions looked like the original in the OOMapper on the screen.

**Comment by dg0yt** [comment]
> The described colour change happened when exporting File > Export as... > PDF > Color mode: Default. I analysed the resulting colours with an eyedropper tool - they had changed significantly. Also when printed them with 2 different printers - colours didn't look like on the screen and were hardly distinguishable from each other.

I don't dispute your observations as such. I just try to make clear that RGB values are not wrong in the PDF: They are just not well-defined without a color space.

> When I chose Color mode: Device CMYK instead the colours were exactly the same in the resulting PDF file and also the printed versions looked like the original in the OOMapper on the screen.

It is good to know that my code works so well for you. Even the QGIS community has some interested in that code ;-)

Still, the result is dependant on the device. Linux and macOS use PDF even as an internal format in the printing system. We have no control over (undesired) conversions from CMYK to RGB being made in the printing system before passing data to the printer or driver.

There is also no natural match between the CMYK colors of the map drawing and the RGB colors of plain templates such as logos of sponsors.

**Comment by puzzlepaint** [comment]
> > so I agree with request to rename "Default" to "Device RGB (default)".
> 
> This was not requested, but is a new proposal.

I would agree with the intention of both proposals to clarify that "Default" means that RGB colors are used (irrespective of the exact formulation).

**Comment by eolmapper** [comment]
Up,
Can we at least put the CMYK mode first, because it is this mode that allows us to print a map correctly most of the time for non-specialist users.
